types/react-native/index.d.ts

```diff
@@ -3024,7 +3103,7 @@ export class PickerIOSItem extends React.Component<PickerIOSItemProps> {}
  */
```

Comment:
Above the PickerIOSItemProps should have an optional `textColor` that is a `ProcessedColorValue` to be parallel with the Flow which is:

```
type RCTPickerIOSItemType = $ReadOnly<{|
  label: ?Label,
  value: ?(number | string),
  textColor: ?ProcessedColorValue,
|}>;
```